apps/web/pages/api/v1/client/environments/[environmentId]/events/index.ts

```diff
@@ -15,10 +15,10 @@ export default async function handle(req: NextApiRequest, res: NextApiResponse)
   }
   // POST
   else if (req.method === "POST") {
-    const { sessionId, eventName, properties } = req.body;
+    const { personId, eventName, properties } = req.body;
```

Comment:
It seems wrong that the sessionId is just replaced with the personId. These are the very old endpoints in the pages directory of formbricks-js < 1.0. Maybe we just remove all of them and force people to the update?
Not sure about this, but slowly it's becoming hard to maintain endpoints so old.